packages/front-end/components/Settings/EncryptionToggle.tsx

```diff
@@ -41,12 +41,10 @@ export default function EncryptionToggle({ form, showUpgradeModal }: Props) {
             />
           </div>
           <div
-            className="col-md-9 text-gray text-right"
+            className="col-md-9 text-gray text-right pt-2"
             style={{ fontSize: 11 }}
           >
-            Only supported when using our Javascript or React SDKs.
-            <br />
-            Requires changes to your implementation.{" "}
+            May require changes to your implementation.{" "}
```

Comment:
It definitely requires changes to their implementation

Reply:
I struggled with this one, basically because if the user had already made the necessary changes to their implementation, it felt a bit strong to claim that they still require changes. But I can change this...